### PR TITLE
Hamburger menu animation

### PR DESCRIPTION
**Pull request by giladt**
Added animation to the menu button in mobile view. [issue](https://www.notion.so/curvelabs/Add-Animation-to-the-Hamburger-menu-on-Mobile-3bc1e9a355fc412987e8d8518a8f6468)

humburger menu animation

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -15,25 +15,32 @@
...
-      <div class="mobileMenuContainer ${showingMobileMenu ? 'showingMobileMenu' : ''}">
-        <div class="menu">
-          <networkfeedback></networkfeedback>
-          <div class="mobileConnectButton">
-            <connectbutton hide-balances></connectbutton>
-          </div>
-          <div class="menuitems">
-            <header-buttons vertical on-navigate.call="onNavigate()"></header-buttons>
-          </div>
+        <div class="mobilemenuButton ${showingMobileMenu ? 'showingMobileMenu' : ''}" click.delegate="toggleMobileMenu()">
```
Comment (dkent600) [inline src/app.html:18]: whaaa?  Why is this needed?  If absolutely necessary, please add a comment in the code.

File: src/app.scss [hunk 2]
```diff
@@ -66,8 +64,13 @@
...
   }
 
+  .appHeader, .logo {
+    top: unset;
+    z-index: 998;
+  }
+
   .mobileMenuContainer {
-    z-index: 998; // stay below modal screen
+    z-index: 997; // stay below modal screen
```
Comment (dkent600) [inline src/app.scss:73]: just curious:  why decrease by one?

**Review by dkent600 — CHANGES_REQUESTED** [review]
@giladt There are a ton of changes here that are impossible to review.  Can you please, at this point where we absolutely must limit the number of code changes, and must be carefully reviewing all code changes, minimize the number of code changes and resist the temptation to reorganize code?  I understand you moved some HTML around, which perhaps necessitated a matching reorg of CSS.  So just sayin'...

File: src/app.html (hunk 1 above)
Comment (giladt) [reply src/app.html:18]: Here I've added a new hamburger icon element, which is easier to animate into an `X` when the menu is open.

File: src/app.html [hunk 3]
```diff
@@ -15,25 +15,32 @@
...
       </div>
-      <div show.to-view="!showingMobileMenu" class="mainContent">
+      <div class="mainContent">
         <router-view></router-view>
       </div>
-      <div show.to-view="!showingMobileMenu" class="appFooter">
+      <div class="appFooter">
         <footer></footer>
       </div>
+      <div if.bind="showingMobileMenu" class="animateMobileMenuBackground au-animate">
```
Comment (giladt) [inline src/app.html:31]: I had to move this `mobileMenuContainer` out from the grid, to the parent element in order to animate it on top of the page.

File: src/app.scss (hunk 2 above)
Comment (giladt) [reply src/app.scss:73]: I guess it was a left over from some experiment I've done. Changed it back.

**Review by dkent600 — APPROVED** [review]
